lucene/core/src/test/org/apache/lucene/search/TestPointQueries.java

```diff
@@ -1903,9 +1903,6 @@ public void testRangeOptimizesIfAllPointsMatch() throws IOException {
       upperBound[i] = value[i] + random().nextInt(1);
     }
     Query query = IntPoint.newRangeQuery("point", lowerBound, upperBound);
-    Weight weight = searcher.createWeight(query, ScoreMode.COMPLETE_NO_SCORES, 1);
-    Scorer scorer = weight.scorer(searcher.getIndexReader().leaves().get(0));
-    assertEquals(DocIdSetIterator.all(1).getClass(), scorer.iterator().getClass());
```

Comment:
The iterator is not wrapped when the score mode is set to `COMPLETE_NO_SCORES` so you don't need to change this assertion anymore ?